ci/pipeline.yml

```
@@ -1937,13 +2180,22 @@ resources:
       json_key: ((bosh_director_oss_ci_service_account_json))
       versioned_file: "bosh-dev-release.tgz"
 
+   # ci/tasks/make-candidate.sh passes--timestamp-version so the captured integer is
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)